tools/clang/test/Dxil2Spv/passthru-cs.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: %dxil2spv
+; RUN: %dxil2spv | %FileCheck %s
```

Comment:
If the filecheck if new enough, you may want to use `--match-full-lines`: https://llvm.org/docs/CommandGuide/FileCheck.html#:~:text=By%20default%2C%20FileCheck%20allows%20matches,affected%20by%20this%20option!).
If the copy in this repo doesn't have this mode, it's also fine as-is.

Reply:
Fails with "Invalid argument", unfortunately